llvm/test/tools/sycl-post-link/sycl-external-funcs/split-and-lower-esimd.ll

```diff
@@ -0,0 +1,29 @@
+; RUN: sycl-post-link -split=auto -split-esimd -lower-esimd -O0 -S %s -o %t.table
+; RUN: FileCheck %s -input-file=%t_esimd_0.ll
+
+; This test checks that unreferenced functions are not dropped
+; from the module and ESIMD lowering happens for them as well.
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64-unknown-linux-sycldevice"
+
+@__spirv_BuiltInGlobalInvocationId = external dso_local local_unnamed_addr addrspace(1) constant <3 x i64>, align 32
```

Comment:
This is pretty small test, so this is just a nit: it is more illustrative, when resulting IR line is located right after the source one.